scripts/Engineering/gui/engineering_diffraction/tabs/focus/view.py

```diff
@@ -46,6 +49,18 @@ def set_focus_button_enabled(self, enabled):
     def set_plot_output_enabled(self, enabled):
         self.check_plotOutput.setEnabled(enabled)
 
+    def set_cropping_widget_hidden(self):
+        self.widget_cropping.hide()
+
+    def set_cropping_widget_visible(self):
+        self.widget_cropping.show()
```

Comment:
You can use `setVisible(bool)` so you only have one function for toggling the widget's visibility